docs/DatabaseCatalogAPI.yaml

```diff
@@ -367,10 +367,62 @@ components:
           properties:
             locations:
               $ref: "#/components/schemas/Locations"
-        - type: object
-          properties:
             latest_dataset:
               $ref: "#/components/schemas/LatestDataset"
+            gtfs_rt_feeds:
```

Comment:
It seems to me that the feed response is becoming quite _fat_
Have you considered using the search or gtfs_rt_feeds endpoint to obtain this info?
It would require multiple calls to the API to obtain all the data, but don't we have to draw the line somewhere?

Reply:
The `search` endpoint uses the GtfsFeed entity in the response, but there is no plan to add it as a parameter.  Adding a parameter to the search might be less intuitive to discover than a field within the gtfs entity. 
We can filter by feed_id in the `gtfs_rt_feeds`, but this will be a different issue than the one we are implementing, see #420.  Regarding multiple calls, the UI at #467 will have to do "multiple" anyway, as the GTFS_RT screen will have to call the `gtfs_feeds/{id}` to retrieve the "related" realtime feeds.

Reply:
But my main point is that the gtfsfeed response is getting quite big. 
You don't think that adding an argument to /v1/gtfs_rt_feeds like schedule_feed=mdb-1 would be better?
It would require calling this endpoint for every schedule feed we want to list, which is not great I guess.

Reply:
The requirement is to get the real-time feeds given a gtfs ID. Adding an argument will do, but it will be prompt to errors from the consumer side as the filtering will apply all the possible arguments. For more strict implementation, if the main concern is the size of the response, I would prefer to add an endpoint to express only the related feeds with path: `v1/gtfs_feeds/{id}/gtfs_rt_feeds`. This can return the complete real-time feed structure.

Reply:
The way you propose is fine. To me it's more important to decide if we should return this info with the GtfsFeed response or require an extra call to the API to obtain it. I am leaning towards having the extra call.

Reply:
> The way you propose is fine. To me it's more important to decide if we should return this info with the GtfsFeed response or require an extra call to the API to obtain it. I am leaning towards having the extra call. But I will leave the final decision to you.

In case of a new endpoint `v1/gtfs_feeds/{id}/gtfs_rt_feeds`, this will be a new call as it will return a full GTFS_rt instead of GTFS. So there is no additional entity to add to the model.

Reply:
@jcpitre Added a new endpoint and removed all the previous entities. It looks cleaner now!